kds/kds.go

```diff
@@ -671,7 +786,20 @@ func parseTCBURL(u *url.URL) (uint64, error) {
 	if err != nil {
 		return 0, fmt.Errorf("invalid AMD KDS TCB arguments: %v", err)
 	}
-	return uint64(tcb), err
+	return tcb.TCB, err
+}
+
+// ProductLineToTCBVersion maps the product line to the TCB version they use
+// in V{C,L}EK x509 extensions
+func ProductLineToTCBVersion(productLine string) (uint8, error) {
+	switch productLine {
```

Comment:
Not sure where this is being called, but we definitely want to normalize this.
```suggestion
	switch strings.Title(productLine) {
```

Reply:
I'm using `cases.Title` instead as `strings.Title` appears to be deprecated.